e2e/icons-react/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -18,6 +18,10 @@ Array [
   "AddFilled",
   "AddLarge",
   "AgricultureAnalytics",
+  "Ai",
```

Comment:
This may be a dumb question, but does it matter that this says "Ai" with a lowercase "i" rather than an uppercase "I" like "AI". I just noticed in other places it is referred to as "AI" for this particular svg.

Reply:
For this one since it stands for artificial intelligence the "I" should probably be capitalized.

Reply:
@laurenmrice see my other comment but I changed my mind and think it's better to leave em alone

Reply:
Okay cool!